src/App.js

```diff
@@ -24,7 +34,7 @@ function App() {
     }
   });
   return (
-    <div className="App font-primary">
+    <div className="App font-primary bg-gray-50 relative p-6 pt-40 min-h-screen">
```

Comment:
- [ ] Kindly remove this new styles because its breaking the login, register, add new plane pages:


https://user-images.githubusercontent.com/95412943/230764497-c4375461-095d-4379-80c7-aeb26db077c3.mp4